int16[] velocity_cmd
time stamp

